requirements.txt

```diff
@@ -16,7 +16,7 @@ pydocstyle
 pyflakes
 pylint
 pylint-django<2.0
-ruff
+ruff>=0.0.237
```

Comment:
I think this change should be added to `setup.py` as well: https://github.com/sscpac/statick/blob/main/setup.py#L59